packages/client/src/pages/vuex.vue

```diff
@@ -0,0 +1,102 @@
+<script setup lang="ts">
+import { getInspectorState, getInspectorTree, onEditInspectorState, onInspectorStateUpdated, onInspectorTreeUpdated } from '@vue/devtools-core'
+import { parse } from '@vue/devtools-kit'
+import type { InspectorState, InspectorTree } from '@vue/devtools-kit'
+import { Pane, Splitpanes } from 'splitpanes'
+
+const inspectorId = 'vuex'
+
+const selected = ref('')
+const tree = ref<Required<InspectorTree>[]>([])
+const state = ref<{
+  state: InspectorState[]
+  getters: InspectorState[]
+}>({
+  state: [],
+  getters: [],
+})
+
+function getVuexState(nodeId: string) {
+  getInspectorState({ inspectorId: 'vuex', nodeId }).then((data) => {
+    state.value = parse(data!)
+  })
+}
+
+function clearVuexState() {
+  state.value = {
+    state: [],
+    getters: [],
+  }
+}
+
+watch(selected, () => {
+  clearVuexState()
+  getVuexState(selected.value)
+})
+
+createCollapseContext('inspector-state')
+createSelectContext('inspector-state')
+
+const selectVuexTree = (id: string) => {
+  selected.value = id
+}
+
+onDevToolsClientConnected(() => {
+  getInspectorTree({ inspectorId: 'vuex', filter: '' }).then((_data) => {
+    const data = parse(_data!)
+    tree.value = data
+
+    if (!selected.value && data.length) {
+      selected.value = data[0].id
+      getVuexState(data[0].id)
+    }
+  })
+
+  onInspectorTreeUpdated((data) => {
+    if (!data?.data.length || data.inspectorId !== inspectorId)
+      return
+    tree.value = data.data as unknown as Required<InspectorTree>[]
+    if (!selected.value && data.data.length) {
+      selected.value = data.data[0].id
+      getVuexState(data.data[0].id)
+    }
+  })
+
+  onEditInspectorState((payload) => {
+    if (payload.inspectorId !== inspectorId)
+      return
+    getVuexState(selected.value)
+  })
```

Comment:
It is a workaround for updating the status of the Vuex in the devtool.

The Vuex side doesn't notify devtool side when the status is modified by devtool.

Reply:
It should but Vuex is not under maintenance.

So we are using this workaround for updating the status of vuex in devtool.

Reply:
I see it's been moved into the applet now, and I think we need to move it back to `client/vuex.vue`. The UI of the applet is then abstracted into a common template for all Vue devtools plugins to use.

Reply:
@webfansplz 
Do you mean later we will move all the modules under `module/` to `client/*` and only leave the common APIs in the `applet`. (Of course, the modules in `client/` will be also built on the API of the applet).

And so at this time, we can move the `module/vuex` to `client/` first?

Reply:
@Azurewarth0920 No, I mean we are in the applet Vuex module implements as far as possible same with the other module, then the similar to the following these differences in ` client/pages/SRC/Vuex vue `

```
  onEditInspectorState((payload) => {
    if (payload.inspectorId !== inspectorId)
      return
    getVuexState(selected.value)
  })
```

Reply:
Oh, I see,
So should I move the whole folder `modules/vuex` to `@client/pages` ? Or just move the `vuex/components/store/index.vue` to `@client/pages` ?

Reply:
Let's keep it that way for now, and I'll adjust them later!